backend/services/workflows/app/worker/http.go

```diff
@@ -57,9 +57,7 @@ func processHTTPTask(
 		form := url.Values{}
 		for key, value := range httpTask.FormData {
 			key = ps.ProcessJobString(key)
-			key = ps.MaybeExecuteGoTemplate(key)
 			value = ps.ProcessJobString(value)
-			value = ps.MaybeExecuteGoTemplate(value)
```

Comment:
how can it be unused, when it is used here?

Reply:
There are no Go template directives (`{{ ... }}`) in any of the workflow definitions.

Reply:
I know that, but the commit message is misleading. that was added for some purpose, right? although I do not remember which.

Reply:
I agree with @merlin-northern, this is a feature we have which we are removing because we are not currently using it.